SoftLayer/CLI/subnet/edit_ip.py

```diff
@@ -0,0 +1,27 @@
+"""Edit ip note"""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--ip-address', required=True,
+              help='Assume the ipAddress to set the note.')
+@click.option('--note', help="set ip address note of subnet")
+@environment.pass_env
+def cli(env, identifier, ip_address, note):
```

Comment:
for this I don't think we nee the `--ip-address` option. Just use `identifier`, and it can be either an IP string, or an IP id.

Something like this.
```
    ip_id = None
    if str.isdigit(identifier):
        ip_id = identifier
    else:
        ip_object = mgr.get_ip_by_address(identifier)  
        ip_id = ip_object.get('id')
    mgr.set_subnet_ipddress_note(ip_id, data)
```

 mgr.get_ip_by_address(identifier)  Should make a call to [SoftLayer_Network_Subnet_IpAddress::getByIpAddress()](https://sldn.softlayer.com/reference/services/SoftLayer_Network_Subnet_IpAddress/getByIpAddress/)